app/views/posts/index.html.erb

```diff
@@ -1,21 +1,28 @@
-<p id="notice"><%= notice %></p>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Bootstrap Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
 
-<h1>Posts</h1>
-
-<%= link_to 'New Post', new_post_path %>
-
-<table border="1" class="postsTable">
-  <thead>
-    <tr>
-       <th><%= sortPosts "PosterID" %></th>
-      <th><%= sortPosts "poster_email" %></th>
-      <th><%= sortPosts "Description" %></th>
-      <th><%= sortPosts "updated_at" %></th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
+<div class="container">
+  <h2>Posts</h2>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Poster ID</th>
+        <th>Poster email</th>
+        <th>Description</th>
+        <th>Updated</th>
```

Comment:
Also Oliver made the columns sortable, to avoid having to make repeated changes please pull master before pushing.